### ISSUE TITLE
Add ``add_ancestral`` option to mutate cli

### ISSUE THREAD

**Issue by jeromekelleher** [tskit-dev/msprime#1549]
After #1548 is merged, we should update the CLI so that we can reliably add mutations to an existing tree sequence with mutations.

We can then remove the xfails from the CLI tests.

We should also improve the error reported without ``add_ancestral`` though, as this is obscure at the moment

cc @winni2k

**Comment by winni2k** [comment]
I see #1548 has been merged. I'll try and get to this by the end of this weekend.

**Comment by jeromekelleher** [comment]
Hooray, thanks @winni2k !

**Comment by petrelharp** [comment]
Whoops - this should wait on #1553. (It'd be nice to have your input on that question, though, @winni2k ?)

**Comment by jeromekelleher** [comment]
No longer relevant.
